cirq/experiments/__init__.py

```diff
@@ -16,3 +16,6 @@
     build_entangling_layers,
     cross_entropy_benchmarking,
 )
+
+from cirq.experiments.fidelity_estimation import (
+    estimate_circuit_fidelity,)
```

Comment:
This name needs to be more specific to the technique being used, as there are many many ways that this could be done. For example, whether or not you are using a logarithmic score (e.g. average Bayes points lost) vs a linear score.

Reply:
Renamed to `compute_linear_xeb_fidelity()`.